notebooks/04-dask-xarray.ipynb

```diff
@@ -4961,8 +4974,23 @@
   },
   {
    "cell_type": "markdown",
-   "id": "8ce5b8a3-cea9-4d04-bac0-dd574da5877d",
+   "id": "889e9971-43eb-4837-8d13-e560de21f908",
    "metadata": {},
+   "source": [
+    "### TODO: compute, persist, and load\n",
+    "\n",
+    "For convenience Xarray also provides `.compute` and `.persist`. These will return an Xarray object:\n",
```

Comment:
This is my only big suggestion. May want to cover the differences between these three. It seems to be a common question.

Reply:
Thanks for pointing this out. Yup! I completely forgot about that. Will add a paragraph about them three of them.